torax/_src/geometry/tests/standard_geometry_test.py

```diff
@@ -78,6 +79,20 @@ def test_build_geometry_from_eqdsk(self, geometry_file):
     config = geometry_pydantic_model.EQDSKConfig(geometry_file=geometry_file)
     config.build_geometry()
 
+  @pytest.mark.skipif(
+      importlib.util.find_spec('imas') is None,
+      reason='IMAS-Python optional dependency'
```

Comment:
having understood better now, `imas-python` is a required dep and so we can remove this `skipif`